js/checklist.js

```diff
@@ -56,6 +56,7 @@ CheckListWidget.prototype.execute = function() {
     this.makeChildWidgets();
     this.renderChildren(domNode);
 
+    //  If this is the clear all button … 
```

Comment:
Got it. I have replaced it with /* add event listeners */ in the upcoming version – the following if-statement speaks for itself.